src/Playwright.Tests/Attributes/PlaywrightTestAttribute.cs

```diff
@@ -52,17 +55,6 @@ public PlaywrightTestAttribute(string fileName, string nameOfTest)
         TestName = nameOfTest;
     }
 
-    /// <summary>
-    /// Creates a new instance of the attribute.
-    /// </summary>
-    /// <param name="fileName"><see cref="FileName"/></param>
-    /// <param name="describe"><see cref="Describe"/></param>
-    /// <param name="nameOfTest"><see cref="TestName"/></param>
-    public PlaywrightTestAttribute(string fileName, string describe, string nameOfTest) : this(fileName, nameOfTest)
```

Comment:
This method with this kind of signature we haven't used much, hence removed + modified the tests which used it.